hawk/src/main/java/com/orhanobut/hawk/Hawk.java

```diff
@@ -18,6 +18,10 @@ private Hawk() {
     // no instance
   }
 
+  public enum Status {
+    NOT_INITIALISED, INITIALISED
```

Comment:
yeah the boolean is enough. I had three statuses before but I've to 2..

@Rainer-Lang agree. #enumsmatter